src/app/index.tsx

```diff
@@ -0,0 +1,17 @@
+import Header from '../widgets/header/index';
```

Comment:
use named exports, not default

Reply:
use import ordering
1. sys lib, low level libs
2. shared
3. entities
4. features
5. widgets/pages/app
6. local/relative paths